<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QSlider" name="sldNFlakes">
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sldDirection">
     <property name="minimum">
      <number>-90</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblBorder">
     <property name="text">
      <string>Border</string>
     </property>
     <property name="buddy">
      <cstring>spbBorder</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="spbMinZ">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblNFlakes">
     <property name="text">
      <string>Number of Flakes</string>
     </property>
     <property name="buddy">
      <cstring>spbNFlakes</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="spbMinScale">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="spbMaxZ">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="bbxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblMaxZ_2">
     <property name="text">
      <string>Max Scale</string>
     </property>
     <property name="buddy">
      <cstring>spbMaxScale</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblMinScale">
     <property name="text">
      <string>Min Scale</string>
     </property>
     <property name="buddy">
      <cstring>spbMinScale</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="sldMinZ">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spbNFlakes">
     <property name="maximum">
      <number>4096</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblAcceleration">
     <property name="text">
      <string>Acceleration</string>
     </property>
     <property name="buddy">
      <cstring>spbAcceleration</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblRandFactor">
     <property name="text">
      <string>Rand Factor</string>
     </property>
     <property name="buddy">
      <cstring>spbRandFactor</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spbDirection">
     <property name="minimum">
      <number>-90</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spbBorder">
     <property name="maximum">
      <number>2048</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDirection">
     <property name="text">
      <string>Direction</string>
     </property>
     <property name="buddy">
      <cstring>sldDirection</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sldAcceleration">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblMinZ">
     <property name="text">
      <string>Min Z</string>
     </property>
     <property name="buddy">
      <cstring>spbMinZ</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSlider" name="sldMinScale">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QDoubleSpinBox" name="spbMaxScale">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="spbAcceleration">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="spbRandFactor">
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="sldBorder">
     <property name="maximum">
      <number>2048</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="sldRandFactor">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSlider" name="sldMaxScale">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSlider" name="sldMaxZ">
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="vspSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblMaxZ">
     <property name="text">
      <string>Max Z</string>
     </property>
     <property name="buddy">
      <cstring>spbMaxZ</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSpaceId">
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="hlySpaceId">
     <item>
      <widget class="QComboBox" name="cbxSpaceId"/>
     </item>
     <item>
      <spacer name="hspSpaceId">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spbNFlakes</tabstop>
  <tabstop>spbAcceleration</tabstop>
  <tabstop>sldDirection</tabstop>
  <tabstop>spbDirection</tabstop>
  <tabstop>spbRandFactor</tabstop>
  <tabstop>spbBorder</tabstop>
  <tabstop>spbMinZ</tabstop>
  <tabstop>spbMaxZ</tabstop>
  <tabstop>spbMinScale</tabstop>
  <tabstop>spbMaxScale</tabstop>
  <tabstop>bbxOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bbxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldDirection</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spbDirection</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbDirection</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldDirection</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldNFlakes</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spbNFlakes</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbNFlakes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldNFlakes</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldBorder</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spbBorder</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbBorder</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldBorder</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
